packages/core/content-manager/server/src/controllers/collection-types.ts

```diff
@@ -422,36 +422,35 @@ export default {
       return ctx.forbidden();
     }
 
-    const permissionQuery = await permissionChecker.sanitizedQuery.publish(ctx.query);
-    const populate = await getService('populate-builder')(model)
-      .populateFromQuery(permissionQuery)
-      .populateDeep(Infinity)
-      .countRelations()
-      .build();
+    const { locale } = getDocumentLocaleAndStatus(body);
 
-    const entityPromises = ids.map((id: any) => documentManager.findOne(id, model, { populate }));
-    const entities = await Promise.all(entityPromises);
+    const documentLocales = await documentManager.findLocales(documentIds, model, {
```

Comment:
Can you call findLocales this way? I dont think it currently supports multiple documentIds

Reply:
It works, but the function's type are not expecting an array, on the [bulkDelete PR](https://github.com/strapi/strapi/pull/20161/files#diff-6d03b8fb4e523d8c3333b4225c5c685874884b96e4b0d35f8f4e878671b32087) I added, I will do the same here